File: Demo/Sources/ViewControllers/Choise/TableCells/ChoiseCellTableViewCell.m
//
//  Globus
//

#import "ChoiseCellTableViewCell.h"
#import "ChoiseViewModel.h"

@interface ChoiseCellTableViewCell ()

@property(nonatomic, weak) IBOutlet UILabel* displayLabel;

@end

@implementation ChoiseCellTableViewCell

+ (NSString*)glb_reuseIdentifier {
    return @"ChoiseCellTableViewCell";
}

- (void)configureWithChoiseViewModel:(ChoiseViewModel*)choiseViewModel {
    _displayLabel.text = choiseViewModel.title;
    _displayLabel.textColor = UIColor.darkGrayColor;
}

@end

File: Demo/Sources/ViewControllers/Choise/TableCells/ChoiseCellTableViewCell.swift
//
//  Globus
//

import Globus

class ChoiseCellTableViewCell: UITableViewCell, GLBNibExtension {
    
    // MARK: - Outlet property
    
    @IBOutlet fileprivate weak var displayLabel: UILabel!
    
    // MARK: - Public func

    public override class func glb_reuseIdentifier() -> String {
        return "ChoiseCellTableViewCell"
    }
    
    public func configure(viewModel: ChoiseViewModel) {
        self.displayLabel.text = viewModel.title;
        self.displayLabel.textColor = UIColor.darkGray
    }
    
    // MARK: - GLBNibExtension
    
    public static func nibName() -> String {
        return "ChoiseCellTableViewCell-Swift"
    }
    
    public static func nibBundle() -> Bundle {
        return Bundle.main
    }
    
}
